docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -749,6 +827,207 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "49a65992",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The bitstring representing the shape of the protein during optimization is:  101100011\n",
+      "The expanded expression is: 1______0_____________________________________________________________________________________________________________________________110001_1____\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = protein_folding_problem.interpret(raw_result=raw_result)\n",
+    "print(\n",
+    "    \"The bitstring representing the shape of the protein during optimization is: \",\n",
+    "    result.turns_sequence,\n",
+    ")\n",
+    "print(\"The expanded expression is:\", result.get_result_binary_vector())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19578a0c",
+   "metadata": {},
+   "source": [
+    "Now that we know which qubits encode which information, we can decode the bitstring into the explicit turns that form the shape of the protein."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "f377810d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The folded protein will have a main turns sequence of: [1, 0, 3, 2, 0, 3]\n",
+      "and the following side turns sequences: [None, None, None, None, None, None, None]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\n",
+    "    f\"The folded protein will have a main turns sequence of: {result.protein_shape_decoder.main_turns}\"\n",
+    ")\n",
+    "print(f\"and the following side turns sequences: {result.protein_shape_decoder.side_turns}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de27cfcd",
+   "metadata": {},
+   "source": [
+    "From this sequence of turns we can get the cartesian coordinates of each of the aminoacids of the protein."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "39402e34",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[['A' '0.0' '0.0' '0.0']\n",
+      " ['P' '0.5773502691896258' '0.5773502691896258' '-0.5773502691896258']\n",
+      " ['R' '1.1547005383792517' '0.0' '-1.1547005383792517']\n",
+      " ['L' '1.7320508075688776' '-0.5773502691896258' '-0.5773502691896258']\n",
+      " ['R' '2.3094010767585034' '0.0' '0.0']\n",
+      " ['F' '1.7320508075688776' '0.5773502691896258' '0.5773502691896258']\n",
+      " ['Y' '1.154700538379252' '1.1547005383792517' '0.0']]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(result.protein_shape_file_gen.get_xyz_file())\n",
+    "result.save_xyz_file()"
```

Comment:
Could you save the save the result in a tmp file - see the Python `tempfile` module. That folder should always be writable for the user something the current folder might not be. You then do not have to worry about already existing etc. (which is why build is failing it seems) Though I honestly wonder about whether that has value here. The print is information that ends up in the xyz, though what is printed is not yet xyz format. If it saved then loaded and the file contents printed too maybe that could work. Otherwise I think the notebook here would work fine with that save_xyz_file() call removed right. And on the name `get_xyz_file` since this does not have the number of atoms and comment part of the file would a better name be `get_xyz_data` since this is the data that will go into the xyz file but its not really an xyz file yet.